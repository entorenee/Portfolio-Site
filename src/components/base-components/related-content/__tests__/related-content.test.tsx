import React from 'react'
import { render } from '@testing-library/react'

import RelatedContent from '..'

describe('<RelatedContent />', () => {
  const props = {
    cards: [
      {
        excerptText: 'This is a sample excerpt from a previous post',
        headlineText: 'Find Great Ways to Test your Components',
        link: {
          text: 'Read More',
          url: 'blog',
        },
      },
      {
        excerptText: 'This is a sample excerpt from a previoust post',
        headlineText: 'Amazing Headline. Come and Read this Article',
        link: {
          text: 'Read More',
          url: 'blog',
        },
      },
    ],
  }

  it('renders correctly', () => {
    const { container } = render(<RelatedContent {...props} />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
