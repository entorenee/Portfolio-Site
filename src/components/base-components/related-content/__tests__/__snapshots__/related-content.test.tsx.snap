// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RelatedContent /> renders correctly 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 0 -1rem 2.4rem -1rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.emotion-0>* {
  margin: 0 1rem;
  max-width: 30rem;
  -webkit-flex: 1 0 30%;
  -ms-flex: 1 0 30%;
  flex: 1 0 30%;
}

.emotion-1 {
  margin-bottom: 1.5rem;
  border: 1px solid #ccc;
  padding: 0.5rem 0.25rem;
  box-shadow: 3px 3px 5px #656E9A;
  padding: 1rem 0.75rem;
}

<div
  class=" emotion-0"
>
  <div
    class="emotion-1"
  >
    <h3>
      Find Great Ways to Test your Components
    </h3>
    <p>
      This is a sample excerpt from a previous post
       
      <a
        href="/blog"
      >
        Read More
      </a>
    </p>
  </div>
  <div
    class="emotion-1"
  >
    <h3>
      Amazing Headline. Come and Read this Article
    </h3>
    <p>
      This is a sample excerpt from a previoust post
       
      <a
        href="/blog"
      >
        Read More
      </a>
    </p>
  </div>
</div>
`;
